src/content-scripts/inject.ts

```diff
@@ -100,8 +101,7 @@ const runContentScript = () => {
     if (document.readyState === 'complete') {
       clearInterval(readyStateCheckInterval);
       const port = browser.runtime.connect();
-      // @ts-expect-error
-      const extConnection = new BrowserRuntimeConnection({ port });
+      const extConnection = new BrowserRuntimeConnection({ port, debug: false });
```

Comment:
Can you explain this change? What is pros/cons of this?

Reply:
This is actually a bug on sdk side that I'll fix in 12.1.0: `debug` shouldn't be required. From your side `@ts-expect-error` is an overkill because it can ignore a more significant TS error, its usage should be limited.